CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# Products
+
+/products/rhel*/product.yml @dahaic @evgenyz @ggbecker @jan-cerny @Mab879 @marcusburghardt @matejak @matusmarhefka @mildas @vojtapolasek @yuumasato
+
+# Product Specific Profiles
+
+/products/rhel*/profiles/ @dahaic @evgenyz @ggbecker @jan-cerny @Mab879 @marcusburghardt @matejak @matusmarhefka @mildas @vojtapolasek @yuumasato
+/controls/cis_rhel7.yml @dahaic @evgenyz @ggbecker @jan-cerny @Mab879 @marcusburghardt @matejak @matusmarhefka @mildas @vojtapolasek @yuumasato
```

Comment:
@matejak GitHub supports using teams in the CODEOWNERS file. It can be useful to create a "Red Hat Employees" team and use the team instead of listing the people repeatedly. I can't setup that because I'm not an admin.

Reply:
@jan-cerny I have created a "Red Hatters" team.